azurerm/internal/services/compute/linux_virtual_machine_resource.go

```diff
@@ -1089,19 +1089,21 @@ func resourceLinuxVirtualMachineDelete(d *schema.ResourceData, meta interface{})
 		return fmt.Errorf("retrieving Linux Virtual Machine %q (Resource Group %q): %+v", id.Name, id.ResourceGroup, err)
 	}
 
-	// ISSUE: XXX
-	// shutting down the Virtual Machine prior to removing it means users are no longer charged for the compute
-	// thus this can be a large cost-saving when deleting larger instances
```

Comment:
can we add this comment back in (but move it below) - details here: https://github.com/terraform-providers/terraform-provider-azurerm/issues/4920

Reply:
Fine for me to add a comment but I try to rewrite the comment because you don't save money by the vm itself only for the disk etc. (https://docs.microsoft.com/en-us/azure/virtual-machines/states-lifecycle)